api/src/services/users/users.test.ts

```diff
@@ -174,60 +184,179 @@ describe('user queries', () => {
-      id: scenario.user.one.id,
-    })) as User
-    const result = await user({ id: original.id })
 
-    expect(result).toEqual(null)
+    scenario('creates a user', async (scenario: StandardScenario) => {
+      const passageUser = { id: 'new-id-1' }
+      mockPassageUser.create.mockReturnValue(passageUser)
+      mockPassageUser.activate.mockReturnValue(passageUser)
+
+      mockCurrentUser({
+        id: scenario.user.one.id,
+        email: scenario.user.one.email,
+        roles: ['USDR_ADMIN'],
+      })
+
+      const result = await createUser({
+        input: {
+          email: 'uniqueemail2@test.com',
+          name: 'String',
+          agencyId: scenario.agency.one.id,
+          role: 'USDR_ADMIN',
+        },
+      })
+
+      expect(result.passageId).toEqual('new-id-1')
+    })
+
+    scenario(
+      'updates a user - deactivate',
+      async (scenario: StandardScenario) => {
+        mockCurrentUser({
+          id: scenario.user.one.id,
+          email: scenario.user.one.email,
+          roles: ['USDR_ADMIN'],
+        })
+        const original = (await user({ id: scenario.user.one.id })) as User
+        const result = await updateUser({
+          id: original.id,
+          input: {
+            email: original.email,
+            name: original.name,
+            role: original.role,
+            agencyId: original.agencyId,
+            isActive: false,
+          },
+        })
+
+        expect(result.isActive).toEqual(false)
+        expect(result.passageId).toEqual(null)
+        expect(mockPassageUser.delete).toHaveBeenCalledWith(original.passageId)
+      }
+    )
+
+    scenario(
+      'updates a user - activate',
+      async (scenario: StandardScenario) => {
+        const passageUser = { id: 'new-id-1' }
+        mockPassageUser.create.mockReturnValue(passageUser)
+        mockPassageUser.activate.mockReturnValue(passageUser)
+
+        mockCurrentUser({
+          id: scenario.user.one.id,
+          email: scenario.user.one.email,
+          roles: ['USDR_ADMIN'],
+        })
+        const original = (await user({ id: scenario.user.inactive.id })) as User
+        const result = await updateUser({
+          id: original.id,
+          input: {
+            email: original.email,
+            name: original.name,
+            role: original.role,
+            agencyId: original.agencyId,
+            isActive: true,
+          },
+        })
+
+        expect(result.isActive).toEqual(true)
+        expect(result.passageId).toEqual('new-id-1')
+        expect(mockPassageUser.create).toHaveBeenCalledWith({
+          email: original.email,
+        })
+      }
+    )
+
+    scenario('deletes a user', async (scenario: StandardScenario) => {
+      mockCurrentUser({
+        id: scenario.user.one.id,
+        email: scenario.user.one.email,
+        roles: ['USDR_ADMIN'],
+      })
+      const original = (await deleteUser({
+        id: scenario.user.one.id,
+      })) as User
```

Comment:
```suggestion
      const original = scenario.user.one
```

Reply:
leaving call to delete
```suggestion
      await deleteUser({
        id: scenario.user.one.id,
      }))
          expect(mockPassageUser.delete).toHaveBeenCalledWith(scenario.user.one.passageId)
```